guides/lightdash-mcp.mdx

```diff
@@ -0,0 +1,362 @@
+---
+title: "Lightdash MCP"
+---
+
+<CardGroup cols={2} iconType="solid" horizontal>
+  {" "}
+  <Card title="Cloud Pro" icon="bolt" /> <Card
+    title="Cloud Enterprise"
+    icon="rocket"
+  />{" "}
+</CardGroup>
+
+The [Model Context Protocol (MCP)](https://modelcontextprotocol.io/) enables AI assistants (e.g. ChatGPT, Claude) and custom agents to directly interact with your Lightdash data. This integration allows MCP clients to explore your data models, search for metrics and dimensions, and provide data-driven insights - all through natural conversation. You can use MCP with existing AI assistants or integrate it into your own custom agents and automated workflows. MCP uses secure OAuth authentication and respects all your existing access controls, ensuring data remains protected.
+
+With MCP, your AI assistant becomes a data analyst that can:
+
+- Browse and understand your data models
+- Find relevant metrics and dimensions
+- Switch between different projects seamlessly
+- Respect your data governance and access controls
+
+<Info>
+  MCP respects all your existing Lightdash permissions and user attributes. MCP
+  clients can only access the data that your user account has permission to
+  view.
+</Info>
+
+## Get started
+
+Setting up MCP is quick and straightforward. You can connect your AI assistant to your Lightdash instance in just a few minutes.
+
+### Prerequisites
+
+- A Lightdash Cloud Pro or Enterprise account with MCP enabled
+- An MCP-compatible AI assistant (e.g., Claude.ai, Claude Desktop, ChatGPT)
+
+### Installation
+
+#### Option 1: Claude.ai (Web & Desktop Apps)
+
+Set up MCP in the [Claude.ai](https://claude.ai) web app, and it will automatically sync to your Claude Desktop app after restart.
+
+<Accordion title="Setup instructions">
+
+1. **Access Settings in Claude.ai**
```

Comment:
I would add a short description to each of these images for better SEO + integration with other mcp tools

Reply:
@tatianainama can you elaborate?

Reply:
sure! so each of these steps is described with an image. If we could add a supporting text to them like:

```
Navigate to your profile menu (bottom-left corner) > Settings > Connectors.
```

Like an 'alt' for the image to give more context to search engines + other ai tools that might be used to scrape information, but just a thought!

Reply:
sounds good!